src/status_im/ui/screens/profile/subs.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im.ui.screens.profile.subs
+  (:require [re-frame.core :refer [reg-sub subscribe]]
+            [status-im.ui.screens.profile.db :refer [account-profile-keys]]))
+
+(reg-sub :my-profile/get
+  :<- [:get-current-account]
+  :<- [:get :my-profile/profile]
+  (fn [[current-account profile] [_ k]]
+    (or (get profile k)
+        (get current-account k))))
+
+(reg-sub :my-profile.drawer/get
+  :<- [:get-current-account]
+  :<- [:get :my-profile/drawer]
+  (fn [[current-account profile] [_ k]]
+    (or (get profile k)
+        (get current-account k))))
+
+(reg-sub :my-profile/valid-name?
+  :<- [:get :my-profile/profile]
+  (fn [profile]
+    (get profile :valid-name? true)))
+
+(reg-sub :my-profile.drawer/valid-name?
+  :<- [:get :my-profile/drawer]
+  (fn [profile]
+    (get profile :valid-name? true)))
```

Comment:
Same comment.

Reply:
I am not sure how to introduce a function there it is subscriptions, the idea is that it only recomputes when the profile or drawer changes

Reply:
`(or (get ..) (get ..))` is used for 2 subscriptions. This part could be abstracted?

Reply:
I don't think it is worth trading 2 lines for a function and an indirection